pandas/tests/indexes/test_indexing.py

```diff
@@ -82,3 +83,16 @@ def test_contains_with_float_index(self):
         assert 1.1 in float_index
         assert 1.0 not in float_index
         assert 1 not in float_index
+
+
+@pytest.mark.parametrize(
+    "idx", [Index([1, 2, 3]), Index([0.1, 0.2, 0.3]), Index(["a", "b", "c"])]
+)
```

Comment:
could use indices fixture?

Reply:
I don't think so, as the deprecation is only for those 3 index types